Source/BugsnagEvent.m

```diff
@@ -353,12 +361,15 @@ - (instancetype)initWithKSReport:(NSDictionary *)event {
                 self.metadata = [BugsnagMetadata new];
             }
 
-            NSDictionary *deviceMetadata = BSGParseDeviceMetadata(event);
-            [self.metadata addMetadata:deviceMetadata toSection:@"device"];
-
+            // Cocoa-specific, non-spec., device and app data
+            [self.metadata addMetadata:BSGParseDeviceMetadata(event) toSection:@"device"];
```

Comment:
Should we add the device metadata to OOMs too?

Reply:
Out of scope, but no reason not to.  There's also a case for adding `charging` and `battery` to the sentinel file as well; they look to be missing.  Out of scope for this task.